server/routes/reviewer.ts

```diff
@@ -48,6 +48,7 @@ export = function CreateReviewRoutes({ offenderService, reportDetailBuilder, rev
         reports,
         pageData,
         query,
```

Comment:
couldn't use this query as it has `moment`s in which gets converted to a blank value in the query string